users/views.py

```diff
@@ -1,3 +1,34 @@
-from django.shortcuts import render
+import requests, jwt
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+
+from config.settings  import SECRET_KEY, ALGORITHM
+from users.models     import User
+
+class KakaoSignInView(View):
+    def get(self, request):
+        try:
+            kakao_access_token = request.headers["Authorization"]
+            user_information   = requests.get("https://kapi.kakao.com/v2/user/me", headers=({"Authorization": f"Bearer {kakao_access_token}"})).json()
```

Comment:
requests 모듈을 사용하실 때에는 timeout 옵션을 반드시 설정해주셔야 합니다.
그 이유는 timeout option을 추가하지 않으면 서버의 응답을 계속해서 기다리고 있는 문제가 발생 합니다. timeout 옵션을 추가해 주세요.
```suggestion
            user_information   = requests.get("https://kapi.kakao.com/v2/user/me", headers=({"Authorization": f"Bearer {kakao_access_token}"}), timeout=5).json()
```

Reply:
기능 구현에 급급해 해당 모듈에 대한 제대로 된 학습이 되지 않았던 것 같습니다. 따끔한 피드백 감사드립니다!